README.md

```diff
@@ -41,7 +41,15 @@ $ make container IMAGE=your-repo/your-name:here
 To deploy your plugin image to an Ark server:
 
 1. Make sure your image is pushed to a registry that is accessible to your cluster's nodes.
-1. Run `ark plugin add <image>`, e.g. `ark plugin add gcr.io/heptio-images/ark-plugin-example`
+2. Run `ark plugin add <image>`, e.g. `ark plugin add gcr.io/heptio-images/ark-plugin-example`
+
+# Using the plugins
+
+A deployed plugin only means it's available to use, need to modify the configuration to make it effective, to do that:
+    
+1. Run `kubectl edit config <config-name> -n <ark-namespace>` e.g. `kubectl edit config default -n heptio-ark`
+2. Edit the sections: `backupStorageProvider/persistentVolumeProvider`, change the `name` and `config` if needed
+3. Save and quit, the plugin will be used for the next `backup/restore`
```

Comment:
This list could use some copyediting, but no. 2 in the list isn't clear. Under what conditions would you need to change the name and config? Do you need to change both `backupStorageProvider` and `persistentVolumeProvider`? What fields *must* you change? Presumably some, otherwise why do you need to edit in the first place?

Reply:
Say I have written a plugin `ark-blockstore-test`, which is a `blockstore` plugin and name is `test`, to use my plugin, I have to modify the ark config, change the `name` of `persistentVolumeProvider` to `test`, which might be 'aws' before, and save the change. In the no.2 `backupStorageProvider/persistentVolumeProvider`, I mean backupStorageProvider **or** persistentVolumeProvider, not **and**

Reply:
@spartacusX do you want to add another commit to this PR with the requested changes? Or shall i go ahead and merge and fix up in a subsequent PR? Thanks!

Reply:
@Bradamant3 Updated the PR. Please merge, and go ahead fix if anything else. Thanks.